code/game/objects/items/tools/crowbar.dm

```diff
@@ -3,7 +3,6 @@
 	desc = "A small crowbar. This handy tool is useful for lots of things, such as prying floor tiles or opening unpowered doors."
 	icon = 'icons/obj/tools.dmi'
 	icon_state = "crowbar"
-	inhand_icon_state = "crowbar"
```

Comment:
Be explicit with inhand icons that exist please, removing this removes the ability to unit test for missing icons.

Override subtypes that shouldn't have them with `inhand_icon_state = null` instead.

Reply:
this breaks the inhands for the crowbar